pyproject.toml

```diff
@@ -111,12 +112,20 @@ extend-select = [
     "RUF100", # Warn about unused noqa
 ]
 
+[tool.ruff.lint.isort]
```

Comment:
You need to add "I" to the ruff `ruff.lint.select` section to turn on ruff isort.

Reply:
Just working on that. :)